faker/providers/misc/__init__.py

```diff
@@ -70,6 +71,13 @@ def language_code(cls):
         return cls.random_element(cls.language_codes)
 
     @classmethod
+    def uuid(cls):
```

Comment:
On Tue, Jun 2, 2015 at 9:56 AM, Jared Culp <notifications@github.com> wrote:

> Do I need to add any tests/docs?


that'd be great, thanks!